dev-requirements.txt

```diff
@@ -1,3 +1,7 @@
+# necessary to pin this version to <5.0.0 for now
+# due to https://github.com/python/importlib_metadata/issues/406
+# which causes the GitHub action to fail running flake8 with pthon3.7
```

Comment:
python typo - side note should we update our Python versions supported? I think 3.10 and maybe even 3.11 are the current versions.

Reply:
Good idea. I added 3.10:

<img width="1197" alt="Screen Shot 2022-10-04 at 11 32 01 AM" src="https://user-images.githubusercontent.com/34752045/193862149-b3932f4d-f43b-40ba-879b-2e40ee04573a.png">